ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -821,54 +821,190 @@
               fingerprint: false
               only-if-success: false
       - '{name}-{test_name}-for-pull-request':
-          test_name: kind-conformance
```

Comment:
It seems this patch modifies job kind-conformance and kind-networkpolicy, it should add the new jobs rather than modifying existing jobs.